runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -574,6 +576,57 @@ String parseLevelUpWidget(WidgetInfo levelUpLevel)
 		return skillName + "(" + skillLevel + ")";
 	}
 
+	/**
+	 * Parses a WidgetInfo pointing to the second widget of the quest-completed dialog
+	 * into a shortened string for filename usage.
+	 *
+	 * @return Shortened string in the format "Quest(The Corsair Curse)"
+	 */
+	@VisibleForTesting
+	String parseQuestCompletedWidget()
```

Comment:
This should probably just `String parseQuestCompletedText(String text)` which would make it easier to test due to not having the dep on the widget